docs: add the tip for nydus-zran

Desiki-high opened the pull request:
we should tell user the nydus zran image must have the same namespace with the oci image.

Fixs: https://github.com/dragonflyoss/image-service/issues/1223

jiangliu commented on docs/nydus-zran.md:
````diff
@@ -39,6 +39,7 @@ Or you can generate one by `nydusify` tool:
 # Convert the existing OCI image `your-registry.com/node:19.0` to `your-registry.com/node:19.0-nydus-oci-ref`:
 sudo nydusify convert --oci-ref --source your-registry.com/node:19.0 --target your-registry.com/node:19.0-nydus-oci-ref
 ```
+> **Tip**: The nydus zran artifact must have the same namespace with the OCI image
````
zran -> ZRAN?

jiangliu replied:
**Tips**: 
- Nydus ZRAN artifacts must be in the same namespace with the OCI image.

Desiki-high committed:
docs: add the tip for nydus-zran

we should tell user the nydus zran image must have the same namespace with the oci image

Signed-off-by: Desiki-high <ding_yadong@foxmail.com>
